client/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+import PacmanGame from './components/PacmanGame/PacmanGame';
+
+class App extends Component {
+  render() {
+    return (
+      <div class="game">
```

Comment:
Please change this to funcrional component

Reply:
@Shalinit3 removed the App.js file. We are using the App.jsx.